profiles-jar-examples/main-app/src/main/resources/conf/dci-dev-local-cluster.json

```diff
@@ -29,8 +29,9 @@
                 "spark.kubernetes.driverEnv.STORAGE_TYPE": "s3",
                 "spark.kubernetes.driverEnv.S3_ENDPOINT": "http://cortex-minio.cortex.svc.cluster.local:9000",
                 "spark.kubernetes.driverEnv.S3_SSL_ENABLED": "false",
-                "spark.kubernetes.driverEnv.AWS_ACCESS_KEY_ID": "******",
-                "spark.kubernetes.driverEnv.AWS_SECRET_KEY": "*****",
+                "spark.kubernetes.driverEnv.AWS_ACCESS_KEY_ID": "changeme",
+                "spark.kubernetes.driverEnv.AWS_SECRET_KEY": "changeme",
+                "spark.kubernetes.driverEnv.CONN_AWS_SECRET": "changeme",
                 "spark.kubernetes.authenticate.driver.serviceAccountName": "default",
```

Comment:
password

Reply:
didn't really think we cared but yeah `dci-dev` and this repo is publicly accessible.. I'll take it out